File: Bab 8/Exercise/6.1. Pola Char.c
#include <stdio.h>
#include <string.h>

int main(int argc, char const *argv[])
{
	
	char string[99];
	int baris, kolom, spasi, panjang;

	scanf("%s", &string);

	panjang =  strlen(string) - 1;
	spasi = panjang - 1;


	for (baris = panjang; baris >= 0; baris--)
	{

		if (baris < panjang - 1)
		{
			spasi--;
		}

		for (kolom = 0; kolom < spasi; kolom++)
		{
			printf(" ");
		}

		printf("%c\n", string[baris]);
	}

	return 0;
}

File: Bab 8/Exercise/6.1. Pola Char.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		
		String string = "";
		int baris, kolom, panjang, spasi;

		Scanner sc = new Scanner(System.in);

		baris = 0;
		kolom = 0;
		panjang = 0;
		spasi = 0;

		try{

			string = sc.next();

		}catch(Exception e){

		}

		panjang = string.length() - 1;
		spasi = panjang - 1;

		for (baris = panjang; baris >= 0; baris--)
		{

			if (baris < panjang - 1)
			{
				spasi--;
			}

			for (kolom = 0; kolom < spasi; kolom++)
			{
				System.out.print(" ");
			}

			System.out.println(string.charAt(baris));
		}
	}
}